resources/views/adminHome.blade.php

```diff
@@ -227,11 +229,11 @@
       </div>
       <div class="d-flex justify-content-center">
       <div class="row pt-lg-5">
```

Comment:
![image](https://user-images.githubusercontent.com/66113867/159856111-c106844e-5bdb-4dd7-a4f6-27b7686f6ba1.png)
```suggestion
      <div class="row pt-lg-5 pb-lg-5">
```

Reply:
tambahin padding bottom nya @ferry5245 , itu terlalu mepet ke bagian bawah nya, nanti coba kirim ss nya ya fer

Reply:
Okee, aman bang, udh di commit yaaa